tests/test.js

```diff
@@ -24,7 +41,7 @@ test('pino caller works also when adding level', function (t) {
 
   const pinoInstance = pinoCaller(pino(through2(function (chunk, enc, callback) {
     const res = JSON.parse(chunk.toString('utf8'))
-    const regex = /Test.<anonymous> \(\/(.)*tests\/test.js/
+    const regex = /Proxy.fatal \(\/(.)*\/pino\/lib\/levels\.js/
```

Comment:
`Test.<anonymous>` is the correct output of the test.

Reply:
not sure why but when i pulled the codebase the test was immediately failing because of that ...

Reply:
Same as Travis: https://travis-ci.org/pinojs/pino-caller/jobs/614735264?utm_medium=notification&utm_source=github_status

Reply:
it seems that #18  is actually fixing that.